rest/client/interface.go

```diff
@@ -123,4 +123,7 @@ type Communicator interface {
 
 	// Fetch list of distributions evergreen can spawn
 	GetDistrosList(context.Context) ([]restmodel.APIDistro, error)
+
+	// Fetch a User's public keys
+	GetKeysList(context.Context) ([]restmodel.APIPubKey, error)
```

Comment:
I realize the signature above does the same thing, but I think this function should take in a user (or user name) and get their keys, rather than taking in a context and figuring out the user from there

Reply:
I agree. 

Though, really, what the underlying route does is "get current user's keys" which if we change the name/documentation, might make things seem less odd.